synapse/tests/test_cortex.py

```diff
@@ -896,269 +886,272 @@ def onPropDel(node, oldv):
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(1, nodes)
             self.eq(nodes[0][0], ('teststr', 'foobar'))
 
             # Traverse from a edge to :n2 with a join
             # (this is technically a circular query)
             q = 'teststr=foobar -> has <+- teststr'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(2, nodes)
             self.eq(nodes[0][0], ('has', (('teststr', 'foobar'), ('teststr', 'foo'))))
             self.eq(nodes[1][0], ('teststr', 'foobar'))
 
             # Add tag
             q = 'teststr=bar pivcomp=(foo,bar) [+#test.bar]'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(2, nodes)
             # Lift, filter, pivot in
             q = '#test.bar +teststr <- *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(1, nodes)
             self.eq(nodes[0][0], ('pivcomp', ('foo', 'bar')))
 
             # Pivot tests with optimized lifts
             q = '#test.bar +teststr <+- *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(2, nodes)
 
             q = '#test.bar +pivcomp -> *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(2, nodes)
 
             q = '#test.bar +pivcomp -+> *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(3, nodes)
 
             # tag conditional filters followed by * pivot operators
             # These are all going to yield zero nodes but should
             # parse cleanly.
             q = '#test.bar -#test <- *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(0, nodes)
 
             q = '#test.bar -#test <+- *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(0, nodes)
 
             q = '#test.bar -#test -> *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(0, nodes)
 
             q = '#test.bar -#test -+> *'
-            nodes = getPackNodes(core, q)
+            nodes = await getPackNodes(core, q)
             self.len(0, nodes)
 
             # Setup a propvalu pivot where the secondary prop may fail to norm
             # to the destination prop for some of the inbound nodes.
-            list(core.eval('[ testcomp=(127,newp) ] [testcomp=(127,127)]'))
-            mesgs = list(core.storm('testcomp :haha -> testint'))
+            await alist(core.eval('[ testcomp=(127,newp) ] [testcomp=(127,127)]'))
+            mesgs = await alist(core.storm('testcomp :haha -> testint'))
+
+            # FIXME: warns are different now somehow
```

Comment:
we need to figure out what is going here and fix it (I'll happily take a stab at it) since I can't think of a reason why this would have changed